moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -24,10 +25,21 @@ const LookupSelectComponent = (props) => {
   return (
     <Select
       variant="standard"
-      style={{ minWidth: "8em" }}
       id={props.name}
       value={value || props.defaultValue}
       ref={ref}
+      disableUnderline
+      sx={{
+        minWidth: "8em",
+        padding: theme.spacing(1),
+        // adding a border despite disabling it above because the default border is much lower than other inputs
+        "& .MuiSelect-select": {
+          borderBottom: "1px rgba(0, 0, 0, 0.42) solid",
+        },
+        "& .MuiSelect-select:hover": {
+          borderBottom: "2px #212121 solid",
+        },
+      }}
```

Comment:
not sure if this matters, but i think you can can declare this object outside of the component so that it doesn't get recreated on every render